rulings/cards-rulings.yaml

```diff
@@ -1,6 +1,7 @@
 ---
 100001|.44 Magnum:
-  - Provides only only maneuver each combat, even if the bearer changes. [LSJ 19980302-2]
+  - Provides only one maneuver each combat, even if the bearer changes. [LSJ 19980302-2]
+  - If you use the maneuver provided by this gun during the determine range step, you are committed to use the strike it provides (2R damage). [BCH 20240228]
```

Comment:
If we want to add this, it needs to be added to all the weapons and strikes that provide a maneuver. I'd rather not include dozens of rulings when it's covered by the rulebook. IMHO this should be removed.